src/main/java/com/vi/appointmentservice/api/facade/AgencyFacade.java

```diff
@@ -121,13 +120,17 @@ public void agencyConsultantsSync(AgencyConsultantSyncRequestDTO request) {
           .collect(Collectors.toList());
       membershipsRepository.updateMemberShipsOfUser(calcomUserToConsultant.get().getCalComUserId(), teamIds);
```

Comment:
Should this not be included in the check?

Reply:
it should work fine, as there is foreach loop inside, that iterates over empty collection, so will have no effect